docs/installing-on-gke.md

```diff
@@ -0,0 +1,321 @@
+```
+
+```bash
+# Grant the Service Account user role to allow the account to act as a service account
+gcloud projects add-iam-policy-binding projects/$PROJECT_ID \
+    --role=roles/iam.serviceAccountUser \
+    --member=principal://iam.googleapis.com/projects/$PROJECT_NUMBER/locations/global/workloadIdentityPools/$PROJECT_ID.svc.id.goog/subject/ns/$NAMESPACE/sa/$KSA_NAME
+```
+
+These command grants the following roles to the service account:
+
+* Compute Instance Admin: Allows Turbinia to attach and detach disks from GCP
+VMs for processing.
+* Service Account User: Allows the Turbinia service account to act as this newly
+created service account.
+
+## Step 4: Deploy the OSDFIR Infrastructure Helm Chart
+
+Now it is time to deploy the OSDFIR Infrastructure Helm chart.
+
+The first step is to add the repo and then update to pick up any new changes.
+
+```console
+helm repo add osdfir-charts https://google.github.io/osdfir-infrastructure
+helm repo update
+```
+
+To install the chart, specify any release name of your choice.
+For example, using `my-release` as the release name, run:
+
+```bash
+helm install my-release osdfir-charts/osdfir-infrastructure \
+    --set turbinia.gcp.enabled=true \
+    --set turbinia.gcp.projectID=$PROJECT_ID \
+    --set turbinia.gcp.projectRegion=$REGION \
+    --set turbinia.gcp.projectZone=$ZONE \
+    --set turbinia.serviceaccount.name=$KSA_NAME
+```
+
+The command deploys OSDFIR Infrastructure on the Kubernetes cluster while enabling
+Turbinia GCP integration.
+
+Verify the deployment:
+
+```bash
+kubectl get pods
+```
+
+You should see pods for Timesketch, Turbinia, and Yeti in a Running state. It may
+take a few minutes for all the services to deploy.
+
+### Provisioning shared filestorage (optional)
+
+While this example uses a single node, you might need persistent storage for
+multi-node deployments. To enable persistent storage with a `ReadWriteMany` PVC,
+add the following `--set` flags to your helm install command:
+
+```bash
+--set persistence.storageClass="standard-rwx" \
+--set persistence.accessModes[0]="ReadWriteMany"
+```
+
+This configures OSDFIR Infrastructure to provision a GCP Filestore instance with
+`ReadWriteMany` access mode, which is suitable for multi-node clusters where
+shared storage is required.
+
+## Step 5: Setup dfTimewolf and CLI configs
+
+OSDFIR Infrastructure utilizes dfTimewolf for orchestrating forensic collection
+and processing. dfTimewolf allows you to define "recipes" that specify how data
+should be collected, processed by tools like Turbinia, and exported to platforms
+like Timesketch.
+
+To install dfTimewolf, you'll need to have Python 3.11 or greater, `git`, and
+`pip` installed on your machine. dfTimewolf uses Poetry for simplified dependency
+management. Follow these steps to install dfTimewolf:
+
+```bash
+git clone https://github.com/log2timeline/dftimewolf.git && cd dftimewolf
+pip install poetry
+poetry install && poetry shell
```

Comment:
had an issue on my machine where $HOME/.local/bin was not added to my PATH so poetry was not found.

Reply:
Ack made a note to add that to your PATH in case it shows missing